docs/how_tos/manual_testing.rst

```diff
@@ -8,9 +8,21 @@ Setting up for testing
 
 #. Make or refresh a copy of this repo where it can be seen from inside devstack: ``rsync -sax -delete ./ ../src/event-bus-kafka/`` (and rerun rsync after any edits as needed)
 #. In devstack, start Kafka and the control webapp: ``make dev.up.kafka-control-center`` and watch ``make dev.logs.kafka-control-center`` until server is up and happy (may take a few minutes; watch for ``INFO Kafka startTimeMs``)
+#. Sometimes kafka fails to start, run ``make dev.up.kafka-control-center`` again.
```

Comment:
What failures have you seen?

Reply:
Cannot consistently reproduce it, but sometimes kafka fails to start with below error:

```log
edx.devstack.kafka                | [2023-05-09 08:35:41,604] ERROR Error while creating ephemeral at /brokers/ids/1, node already exists and owner '72058728364572673' does not match current session '72058736877043713' (kafka.zk.KafkaZkClient$CheckedEphemeral)
edx.devstack.kafka                | [2023-05-09 08:35:41,608] ERROR [KafkaServer id=1] Fatal error during KafkaServer startup. Prepare to shutdown (kafka.server.KafkaServer)
edx.devstack.kafka                | org.apache.zookeeper.KeeperException$NodeExistsException: KeeperErrorCode = NodeExists
edx.devstack.kafka                |     at org.apache.zookeeper.KeeperException.create(KeeperException.java:126)
edx.devstack.kafka                |     at kafka.zk.KafkaZkClient$CheckedEphemeral.getAfterNodeExists(KafkaZkClient.scala:2114)
edx.devstack.kafka                |     at kafka.zk.KafkaZkClient$CheckedEphemeral.create(KafkaZkClient.scala:2052)
edx.devstack.kafka                |     at kafka.zk.KafkaZkClient.checkedEphemeralCreate(KafkaZkClient.scala:2019)
edx.devstack.kafka                |     at kafka.zk.KafkaZkClient.registerBroker(KafkaZkClient.scala:100)
edx.devstack.kafka                |     at kafka.server.KafkaServer.startup(KafkaServer.scala:452)
edx.devstack.kafka                |     at kafka.Kafka$.main(Kafka.scala:129)
edx.devstack.kafka                |     at kafka.Kafka.main(Kafka.scala)
edx.devstack.kafka                | [2023-05-09 08:35:41,610] INFO [KafkaServer id=1] shutting down (kafka.server.KafkaServer)
```

It is fixed by restarting.

Reply:
Interesting, thanks. I'll see if I encounter this as well.